library/MedEx/API.php

```diff
@@ -2492,230 +2495,245 @@ public function possibleModalities($appt)
+                        while ($urow = sqlFetchArray($ures)) {
+                            echo "    <option value='" . attr($urow['id']) . "'";
+                            if ($urow['id'] == $defaultProvider) {
+                                echo " selected";
+                            }
+                            echo ">" . text($urow['lname']);
+                            if ($urow['fname']) {
+                                echo ", " . text($urow['fname']);
+                            }
+                            echo "</option>\n";
+                        }
+                        echo "</select>";
+                        ?>
+                    </div>
+                    <div class="form-group col-12 col-md-6">
+                        <label><?php echo xlt('Facility'); ?></label>
+                        <select class="form-control ui-selectmenu-button ui-button ui-widget ui-selectmenu-button-closed ui-corner-all" name="new_facility" id="new_facility" style="width: 95%;">
+                            <?php
+                                $qsql = sqlStatement("SELECT id, name, primary_business_entity FROM facility WHERE service_location != 0");
+                            while ($facrow = sqlFetchArray($qsql)) {
+                                if ($facrow['primary_business_entity'] == '1') {
+                                    $selected = 'selected="selected"';
+                                    echo "<option value='" . attr($facrow['id']) . "' $selected>" . text($facrow['name']) . "</option>";
+                                } else {
+                                    $selected = '';
+                                    echo "<option value='" . attr($facrow['id']) . "' $selected>" . text($facrow['name']) . "</option>";
+                                }
+                            }
+                            ?>
+                        </select>
+                    </div>
+                    <div class="form-group col-12 col-md-6">
+                        <label><?php echo xlt('DOB'); ?></label>
+                        <?php
+                            $DOB = oeFormatShortDate($result_pat['DOB']);
+                        ?>
+                        <span name="new_DOB" id="new_DOB" style="width: 90px;"><?php echo text($DOB); ?></span> -
+                        <span id="new_age" name="new_age"><?php echo text($result_pat['age']); ?></span>
+                    </div>
+                    <div class="form-group col-12 col-md-6">
+                        <label><?php echo xlt('Address'); ?></label>
+                        <div class="row">
+                            <div class="col-12 mb-2">
+                                <input type="text" class="form-control" placeholder="<?php echo xlt('Address'); ?>" name="new_address" id="new_address" value="<?php echo attr($result_pat['street']); ?>">
```

Comment:
placeholder should use xla

Reply:
i'll fix these minor issues since I brought this in already (I actually got confused and was reviewing another PR and then switched to this PR by mistake :) )

Reply:
Ok, Thank you so much.